spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :user do
+    name "Dogbert"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.